components/example-notebook-servers/jupyter-pytorch-full/Makefile

```diff
@@ -15,7 +15,7 @@
 GIT_COMMIT     := $(shell git rev-parse HEAD)
 GIT_TREE_STATE := $(shell test -n "`git status --porcelain`" && echo "-dirty" || echo "")
 
-REGISTRY ?= ghcr.io/kubeflow/notebook-servers
```

Comment:
@thesuperzapper did you saw that https://github.com/orgs/kubeflow/packages?q=kubeflow%2F&tab=packages&q=notebook 

i.e. https://github.com/orgs/kubeflow/packages/container/package/kubeflow%2Fnotebook-servers%2Fjupyter-pytorch-full